src/applications/disability-benefits/526EZ/tests/config/vaFacilities.unit.spec.jsx

```diff
@@ -28,6 +28,9 @@ describe('Disability benefits 526EZ VA facility', () => {
     uiSchema,
     arrayPath,
   } = formConfig.chapters.supportingEvidence.pages.vaFacilities;
+  uiSchema.disabilities.items.treatments.items.treatmentCenterName[
+    'ui:options'
+  ].getOptions = () => Promise.resolve([]);
```

Comment:
I'm going to try to update the `AutoSuggest` component to _not_ debounce this call and maybe do an RFC that recommends how debouncing should be handled. TL;DR debouncing should be wrapped into the call and not something a component does. 

Even with that change, I think it would be necessary to alter this `getOptions` before testing it. Not sure the best way to handle this- if it's typical to edit the form config this way or if there's an easier way we could disable debouncing behavior. This fix in particular is pretty imperative.

Reply:
I can't think of a better way off the top of my head. We probably want to unit test the `getOptions` function separately if we're overriding it, though.

Reply:
Looks like we have tests for `queryForFacilities`. `getOptions` is what the second argument of `autoSuggestUiSchema` get assigned to. https://github.com/department-of-veterans-affairs/vets-website/blob/daa11a751bf30802e3924b0487bd4346542fc726/src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx#L204